autoload/cmake4vim.vim

```diff
@@ -32,10 +32,27 @@ function! cmake4vim#CompleteTarget(arg_lead, cmd_line, cursor_pos) abort
     return join(l:sorted_targets, "\n")
 endfunction
 
+function! cmake4vim#populateDefaultCMakeVariants() abort
+    for build_type in filter( utils#cmake#getDefaultBuildTypes(), "v:val !=# ''" )
```

Comment:
Am I right that you try to remove all empty targets? Is it a real case?

Reply:
`utils#cmake#getDefaultBuildTypes()` returns `[ 'Release', 'Debug', 'RelWithDebInfo', 'MinSizeRel', '' ]`, I just wanted to discard the last, empty, build type.